packages/keyring-snap-client/tsconfig.json

```diff
@@ -3,6 +3,14 @@
   "compilerOptions": {
     "baseUrl": "./"
   },
+  "references": [
+    {
+      "path": "../keyring-api/tsconfig.build.json"
```

Comment:
Did the linter check these files? I think the formatting is not consistent

Reply:
It should since we are using `lint:misc` in the main `lint` command. And running `yarn prettier packages/*/tsconfig*.json -w` does not change anything.

But I agree that it looks somehow inconsistent sometimes 😅